providers/session_state.go

```diff
@@ -47,95 +47,53 @@ func (s *SessionState) String() string {
 
 // EncodeSessionState returns string representation of the current session
 func (s *SessionState) EncodeSessionState(c *cookie.Cipher) (string, error) {
-	if c == nil || s.AccessToken == "" {
-		return s.accountInfo(), nil
-	}
-	return s.EncryptedString(c)
-}
-
-func (s *SessionState) accountInfo() string {
-	return fmt.Sprintf("email:%s user:%s", s.Email, s.User)
-}
-
-// EncryptedString encrypts the session state into a cookie string
-func (s *SessionState) EncryptedString(c *cookie.Cipher) (string, error) {
-	var err error
+	ss := *s
 	if c == nil {
-		panic("error. missing cipher")
-	}
-	a := s.AccessToken
-	if a != "" {
-		if a, err = c.Encrypt(a); err != nil {
+		ss.AccessToken = ""
```

Comment:
Do you need to nil `ss.ExpiresOn` to preserve the existing behaviour?

Probably worth a comment explaining why this is happening too

Reply:
Thanks, addressed in the latest update.